universal_transfer_operator/docs/universal_transfer_operator/bigquery.rst

```diff
@@ -0,0 +1,100 @@
+***************
+Google Bigquery
+***************
+
+Transfer to Google Bigquery as destination dataset
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+User can transfer data to Google Bigquery as destination as from following sources dataset:
+
+#. :ref:`table`
+    - Sqlite
+    - Snowflake
+    - Bigquery
+
+#. :ref:`file`
+    - AWS S3
+    - Google cloud storage
+    - Local
+    - SFTP
+
+Following transfer modes are supported:
+
+1. Transfer using non-native approach
+    Following is an example of non-native transfers between GCS to google bigquery using non-native transfers:
+
+    .. literalinclude:: ../../example_dags/example_universal_transfer_operator.py
+       :language: python
+       :start-after: [START transfer_non_native_gs_to_bigquery]
+       :end-before: [END transfer_non_native_gs_to_bigquery]
+
+2. Transfer using native approach
```

Comment:
Can we remove the approach that we are not supporting yet?

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1861/commits/e6359c247b1ac8547febf6a55ba1c5c15905a97e)